ngraph/src/ngraph/op/non_max_suppression.hpp

```diff
@@ -236,6 +235,60 @@ namespace ngraph
                     clone_with_new_inputs(const OutputVector& new_args) const override;
             };
         } // namespace v4
+
+        namespace dynamic
+        {
+            /// \brief NonMaxSuppression operation
+            ///
+            class NGRAPH_API NonMaxSuppression : public op::v3::NonMaxSuppression
+            {
+            public:
+                static constexpr NodeTypeInfo type_info{"NonMaxSuppression", 99};
```

Comment:
Why do we use 99 here?

Reply:
See my comment above. Actually, I would like to use string "dynamic" but it is not allowed.